ezcontent.profile

```diff
@@ -91,3 +94,17 @@ function ezcontent_themes_installed($theme_list) {
     }
   }
 }
+
+/**
+ * Implements hook_form_alter().
+ */
+function ezcontent_form_alter(array &$form, FormStateInterface $form_state, $form_id) {
+  if ($form_id == 'media_image_add_form' || $form_id == 'media_image_edit_form' || $form_id == 'media_instagram_add_form' || $form_id == 'media_instagram_edit_form' || $form_id == 'media_tweet_add_form' || $form_id == 'media_tweet_edit_form' || $form_id == 'media_video_add_form' || $form_id == 'media_video_edit_form') {
```

Comment:
This will fail in case there are more media bundles are added. Please see if this can be generalised.